### ISSUE TITLE
In the API documentation what is if (window.parent) for?

### ISSUE THREAD

**Issue by Yanal-Yves** [damienbod/angular-auth-oidc-client#335]
Hello,

The API documentation says to implement `onAuthorizationResultComplete` with the following code:
```
private onAuthorizationResultComplete(authorizationResult: AuthorizationResult) {
	if (authorizationResult.authorizationState === AuthorizationState.unauthorized) {
		if (window.parent) {
			// sent from the child iframe, for example the silent renew
			this.router.navigate(['/unauthorized']);
		} else {
			window.location.href = '/unauthorized';
		}
	}
}
```

I don't understand how could the condition `window.parent` be false as it could not be null. 
reference: [Window parent on mozilla doc](https://developer.mozilla.org/en-US/docs/Web/API/Window/parent).

If my understanding is correct the `if (window.parent)` and the else path could be removed.

Could you validate or invalidate my thoughts and fix the API documentation if needed?

**Comment by profet23** [comment]
I agree. The `if` is not needed/not doing anything. Please take a look at the pull request.
